x/rollapp/keeper/fraud_proposal.go

```diff
@@ -0,0 +1,66 @@
+package keeper
+
+import (
+	"context"
+
+	errorsmod "cosmossdk.io/errors"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/dymensionxyz/dymension/v3/x/rollapp/types"
+	"github.com/dymensionxyz/gerr-cosmos/gerrc"
+)
+
+// SubmitRollappFraud handles the submission of a fraud proposal
+// The fraud proposal can be submitted by the gov module
+func (k Keeper) SubmitRollappFraud(goCtx context.Context, msg *types.MsgRollappFraudProposal) (*types.MsgRollappFraudProposalResponse, error) {
```

Comment:
What about double sign?
It won't necessarily incur a hard fork
Is it coming in another PR @omritoptix ?

Reply:
yes.
https://github.com/dymensionxyz/dymension/issues/1465